doc/translation_process.md

```diff
@@ -102,6 +102,6 @@ To create a new language template, you will need to edit the languages manifest
 **Note:** that the language translation file **must end in `.qm`** (the compiled extension), and not `.ts`.
 
 ### Questions and general assistance
-The Bitcoin-Core translation maintainers include *tcatm, seone, Diapolo, wumpus and luke-jr*. You can find them, and others, in the Freenode IRC chatroom - `irc.freenode.net #bitcoin-core-dev`.
+The Bitcoin-Core translation maintainers include *hebasto, laanwj and luke-jr*. You can find them, and others, in the Libera Chat IRC Room - `irc://irc.libera.chat` `#bitcoin-core-dev` ([webchat](https://web.libera.chat/#bitcoin-core-dev)).
```

Comment:
I'm not a translation maintainer, only a coordinator for Ukrainian translation.

Reply:
Ah, sorry. I _am_ an administrator.

Reply:
Fixed

Reply:
Ok, restored

Reply:
After discussing on IRC with @hebasto :
Rather than mention individual people here to contact, we might want to suggest people to ask in the channel. I generally prefer this, at least.
Also, as we now have a GUI repository, they could file an issue there.

Reply:
The "default" way to ask for help is already to connect to IRC or file an issue, so I've removed this sentence completely.